jax/experimental/jax2tf/tests/shape_poly_test.py

```diff
@@ -1847,15 +1855,6 @@ def f2(z, w):  # z: f32[a, 5] w: f32[a + b, 5] -> f32[2*a + b, 10]
     self.assertAllClose(f2(* f1(x, y)), res)
 
 
-def _random_split(key, num, impl):
```

Comment:
This wasn't good enough because we need the flags to stay during lowering too. Added a more general mechanism for override_jax_config_flags.